autodocodec-schema/package.yaml

```diff
@@ -20,6 +20,7 @@ library:
   - aeson
   - autodocodec >=0.0.1.0
   - containers
+  - insert-ordered-containers
```

Comment:
I hope this isn't necessary either...

Reply:
Oh, you're right. It's not necessary. Removed.